src/operators/convert.ts

```diff
@@ -104,7 +108,12 @@ export class ConvertOperator implements Operator {
 
     // if enumerate is set, try to coerce all strings into an equivalent numeric value
     if (enumerate) {
-      const enumerableProps = ConvertOperator.enumerableProperties(data[index]);
+      let enumerableProps = ConvertOperator.enumerableProperties(data[index]);
+      // if ignore properties are set, make sure those are filtered out
+      if (ignore) {
```

Comment:
I'm noticing `ignore` is typed as `any` here. Would it make sense to add this property to `ConvertOperatorOptions` e.g.:

```
export interface ConvertOperatorOptions extends OperatorOptions {
  // ...
  ignore?: string | string[];
  // ...
}
```

Reply:
I totally missed this.  Good catch!